### DISCUSSION TITLE
saved offline/ basic auth ?

### DISCUSSION THREAD

**Discussion by Tyrian81** [mealie-recipes/mealie#1066] [Q&A] [answered]
Are the recipes saved locally/offline ? 
And want to make this externally available for the family , is there some sort of basic auth integrated ?

**Comment by Tukks** [comment, marked as answer]
Recipe are saved in SQLite or PostgreSql database, everything is local
There is basic auth integrated, without account manager and Group manager if you want.
Personally, I created 2 account, one for me et one for my wife with one group (home)
